openhcl/virt_mshv_vtl/src/processor/hardware_cvm/mod.rs

```diff
@@ -431,12 +425,16 @@ impl<T, B: HardwareIsolatedBacking> hv1_hypercall::VtlReturn for UhHypercallHand
 
         self.vp.unlock_tlb_lock(Vtl::Vtl1);
 
+        let hv = &mut self.vp.backing.cvm_state_mut().hv[GuestVtl::Vtl1];
+        if hv.synic.vina().auto_reset() {
+            hv.set_vina_asserted(false).unwrap();
```

Comment:
Is that something we need to defend against? Wouldn't we have hit that earlier in the process?

Reply:
The guest could change protections in a way that makes the page inaccessible at any time. We don't currently lock the VP assist page to prevent this.

It's fine to just turn this into a VP failure, but we shouldn't just panic.